librz/analysis/var.c

```diff
@@ -1314,15 +1335,44 @@ RZ_API void rz_analysis_fcn_vars_cache_fini(RzAnalysisFcnVarsCache *cache) {
 	rz_list_free(cache->stackvars);
 }
 
+static char *sig_from_debuginfo(RzAnalysis *analysis, char *fcn_name, const char *fcn_name_pre, const char *fcn_name_post) {
+	if (!rz_str_startswith(fcn_name, "dbg."))
+		return NULL;
+
+	const char *real_fcn_name = fcn_name + 4;
+	char *key = RZ_STR_ISNOTEMPTY(real_fcn_name) ? rz_str_newf("fcn.%s.sig", real_fcn_name) : NULL;
+	Sdb *sdb = sdb_ns(analysis->sdb, "dwarf", 0);
```

Comment:
For now this is ok, then we should remove it: https://github.com/rizinorg/rizin/issues/3541
This is more long-term work though, after Tricore things and more DWARF fixes are finished